snow/consensus/metrics/height.go

```diff
@@ -3,29 +3,50 @@
 
 package metrics
 
-import "github.com/prometheus/client_golang/prometheus"
+import (
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/ava-labs/avalanchego/utils"
+	"github.com/ava-labs/avalanchego/utils/math"
+)
 
 var _ Height = (*height)(nil)
 
-// Height reports the last accepted height
 type Height interface {
+	Verified(height uint64)
```

Comment:
I called this `Verified` rather than `Issued` because we're moving verification into the consensus code. We'll probably add `Issued` as another tracked height later